src/Microsoft.ML.LightGBM/doc.xml

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<doc>
+  <members>
+    <!--  
+    The following text describes the LightGbm algorithm details.
+    It's used for the remarks section of all LightGbm-based trainers (binary, multiclass, regression, ranking)
+    -->
+    <member name="LightGBM_remarks">
+      <remarks>
+        LightGBM is an open source implementation of boosted trees. For implementation details, please see 
+        <a href='https://lightgbm.readthedocs.io/en/latest/index.html'>LightGBM's official documentation</a>.
```

Comment:
>documentation [](start = 91, length = 13)

Also link to the paper with Guolin and Tom.

Reply:
good point.

---
In reply to: [263945392](https://github.com/dotnet/machinelearning/pull/2886#discussion_r263945392) [](ancestors = 263945392)